### DISCUSSION TITLE
如何提取所需表单数据

### DISCUSSION THREAD

**Discussion by BlueYeah** [alibaba/formily#2127] [Q&A 2.x] [answered]
https://codesandbox.io/s/exciting-tamas-71412

目标: 只提取需要提交的数据，纯展示的数据不作为提交数据。

我现在是自己手写方法提取自己所需数据， 请问有没其他配置方法呢？ 文档我也看过了。

**Comment by janryWang** [comment, marked as answer]
没有配置方法，这个属于业务逻辑，自己过滤掉不需要的字段吧

**Reply by BlueYeah** [reply]
谢谢
